impeller/renderer/backend/vulkan/allocator_vk.cc

```diff
@@ -116,6 +116,7 @@ std::shared_ptr<Texture> AllocatorVK::OnCreateTexture(
   image_create_info.initialLayout = vk::ImageLayout::eUndefined;
   image_create_info.usage = vk::ImageUsageFlagBits::eSampled |
```

Comment:
This makes me think we should perhaps have a recommended usage as on optional on the texture descriptor. For later though.